big-practice/src/js/service/account.service.js

```diff
@@ -0,0 +1,11 @@
+import { ACCOUNTS_API } from '../constants/url-api.constant';
+import { apiService } from './api.service';
+
+export class AccountService {
+  static async getAccount(email, password) {
```

Comment:
getAccount should be come with id, this function name should login or something like that